libvast/vast/system/reader_command.hpp

```diff
@@ -90,6 +90,7 @@ caf::message reader_command(const command& cmd, caf::actor_system& sys,
                                  slice_size, max_events);
       return source_command(cmd, sys, std::move(src), options, first, last);
     };
+    VAST_INFO(reader, "listening for data on", ep.port);
```

Comment:
```suggestion
    VAST_INFO(reader, "listens for data on", ep.port);
```

Maybe also log the IP address and not just the port?